test/com/facebook/buck/jvm/java/intellij/testdata/experimental_project_with_custom_sdks/.buckconfig

```diff
@@ -9,3 +9,5 @@
 [intellij]
   java_library_sdk_names = 1.6 => Java SDK 1.6, \
                            1.8 => Java SDK 1.8
+  android_sdk_name = "Google Inc.:Google APIs:24"
```

Comment:
The same about sdk name, please, replace with the default name "Android API 23 Platform".